tests/unit/coordinator/plugins/test_base.py

```diff
@@ -118,7 +119,14 @@ def named_manifest(filename: str) -> ipc.BackupManifest:
     return make_manifest(some_isoformat, some_isoformat, filename=filename)
 
 
+# Global used but just for test case
+# pylint: disable=global-statement
+COUNT = 0
```

Comment:
No mutable globals, even just for tests.

Reply:
I could use a uuid instead?

Reply:
the functions is called twice, maybe just a parameter that is either 1 or 2 would be enough ?